src/main/java/org/folio/oaipmh/helpers/GetOaiIdentifiersHelper.java

```diff
@@ -117,7 +118,7 @@ private OAIPMH buildListIdentifiers(Request request, JsonObject srsRecords) {
   protected void addResumptionTokenToOaiResponse(OAIPMH oaipmh, ResumptionTokenType resumptionToken) {
     if (oaipmh.getListRecords() != null) {
       oaipmh.getListIdentifiers()
-        .withResumptionToken(resumptionToken);
+        .withResumptionToken(resumptionToken.withExpirationDate(Instant.now().plusSeconds(60)));
```

Comment:
It is a duplicated code. The resumpotonToken, which is passed to the method, is built via the buildResumptionToken method from AbstarctHelper where you already put this attribute. No need to do it again. Please, remove it.